app/helpers/templates_helper.rb

```diff
@@ -27,6 +27,12 @@ def default_params(widget_type)
       text_body: {
         'text_body_html' => 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur nec pretium sem.'
       },
+      thermometer: {
```

Comment:
Does it make sense to pull the thermometer stuff out into its own branch/PR? It seems disconnected from this PR and would probably be trivial to extract.

Just thinking out loud.